recidiviz/ingest/us_ny/us_ny_scraper.py

```
@@ -86,1600 +78,1224 @@
-    and params for the next page of results. Enqueues tasks to scrape both
-    the individual listings and next page of results.
-
-    Args:
-        params: Dict of parameters. This is how DOCCS keeps track of where you
-        are and what you're requesting. Not all of the parameters are
-        understood, and some are frequently empty values.
-
-            First page of results only:
-            first_page: (bool) True / exists only if this is requesting the
-                first page of results. Added in scrape_search_page(), not
-                scraped.
-
-            Subsequent pages of results only:
-            clicki: (string) A request parameter for results, scraped from form
-            dini: (string) A request parameter for results, scraped from the form
-            k01,k02,k03,k04,k05,k06: (strings) Request parameters for results,
-                scraped from form
-            map_token: (string) A request parameter for results, scraped from form
-            next: (string) A request parameter for results, scraped from form
-
-            All results pages:
-            first_name: (string) Given names for inmate query (empty string if
-                surname-only search)
-            last_name: (string) Surname for inmate query (required)
-            k01: (string) A request parameter for results, scraped from form
-            token: (string) DFH_STATE_TOKEN - session info from the form
-            action: (string) The 'action' attr from the scraped form - URL to
-                POST our request to
-            scrape_type: (string) Type of scrape to perform
-            content: (tuple) Search target, stored as a tuple:
-                Background scrape: ("surname", "given names")
-                Snapshot scrape:   ("record_id", ["records to ignore", ...])
-
-    Returns:
-        Nothing if successful, -1 if fails.
-    """
-
-    params = json.loads(params)
+        return None
```

A return strategy of `None` if successful is kind of odd, more so when failure is an integer value. If we have nothing to return in the case of success but we might have some sort of failure to broadcast, I would probably throw an exception and document in that in the method signature so callers can handle it. Alternatively, if there's a reason not to do that, I would update this to return negative values in the case of failure and positive values in the case of success.

---

(I know that for some of my comments in here, I'm commenting on logic that predates your work. But while we're in here cleaning everything up, it's a good idea for us to leave things in a better state!)